src/meetbot-manager.ts

```diff
@@ -96,3 +116,70 @@ export async function killBot(url: string) {
 		throw new Error(`Could not find bot at specified location!`);
 	}
 }
+
+export async function scheduleBotsForMeetings() {
+	let credentials: Credentials | null = null;
+	try {
+		const credentialsFile = fs.readFileSync('credentials.json').toString();
+		credentials = JSON.parse(credentialsFile).installed;
+	} catch (err) {
+		console.log(
+			'Cannot read credentials.json - required to use Google Calendar',
+			err,
+		);
+	}
+	if (!credentials || !process.env.GOOGLE_CALENDAR_NAME) {
+		console.log(
+			'deactivating google calender feature due to missing credentials/calendar name',
+		);
+		return;
+	}
+
+	const calendar = new GoogleCalendar(credentials);
```

Comment:
I don't really like how complex the Google authentication is. This function checks for credentials but the GoogleCalendar implementation also performs some authentication (creating as token). It's become hard to follow how authentication is done so if a token.json file is what all these google APIs use then I think 1 single place should create it and the rest of the code doesn't have to always check if it exists and create if not...not worth doing in this PR but wanted to mention my thoughts about this.

Reply:
I agree to this. I already saw this as a pattern when I was writing the code. This is the first time where the credentials implementation has been duplicated.

Reply:
if it works we can leave it then and address it later as I understand other features already have this issue.